perfkitbenchmarker/disk.py

```diff
@@ -45,6 +93,67 @@ def __init__(cls, name, bases, dct):
     super(AutoRegisterDiskSpecMeta, cls).__init__(name, bases, dct)
 
 
+def WarnAndTranslateDiskTypes(name, cloud):
+  """Translate old disk types to new disk types, printing warnings if needed.
+
+  Args:
+    name: a string specifying a disk type, either new or old.
+    cloud: the cloud we're running on.
+
+  Returns:
+    The new-style disk type name (i.e. the provider's name for the type).
+
+  Raises:
+    ValueError, if name is not a disk type name.
```

Comment:
You're right, this is left over from an older version of the code. I will remove it.